docs/server/source/appendices/run-with-docker.md

```diff
@@ -25,7 +25,7 @@ docker run \
   --interactive \
   --rm \
   --tty \
-  --volume "$HOME/bigchaindb_docker:/data" \
+  --volume $HOME/bigchaindb_docker:/data \
```

Comment:
I wonder why you removed all the double quotes from lines like this. Was it not working?